src/sentry/static/sentry/app/views/releasesV2/list/releasehealth/issuesQuantity.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import {t, tct} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import Link from 'app/components/links/link';
+import Count from 'app/components/count';
+
+import {getReleaseNewIssuesUrl} from '../../utils';
+
+type Props = {
+  orgSlug: string;
+  newGroups: number;
+  projectId: number;
+  releaseVersion: string;
+  isCompact?: boolean;
+};
+
+const IssuesQuantity = ({
+  orgSlug,
+  newGroups,
+  projectId,
+  releaseVersion,
+  isCompact = false,
+}: Props) => {
+  const issuesQuantity = isCompact ? (
+    tct('[count] issues', {
+      count: <Count value={newGroups} />,
+    })
+  ) : (
+    <Count value={newGroups} />
+  );
+
+  if (newGroups !== 0) {
```

Comment:
We had it like this at first but received feedback that it's better to always make it a link. (even if issue count is zero)

Reply:
Oh I see... I will update this part 😉